go/ephemeral/ephemeral_test.go

```diff
@@ -0,0 +1,29 @@
+package libkb
+
+import (
+	"context"
+	"testing"
+
+	"github.com/keybase/client/go/kbtest"
+	"github.com/keybase/client/go/libkb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewDeviceEK(t *testing.T) {
+	tc := libkb.SetupTest(t, "ephemeral", 2)
+	defer tc.Cleanup()
+
+	_, err := kbtest.CreateAndSignupFakeUser("t", tc.G)
+	require.NoError(t, err)
+
+	metadata, err := PublishNewDeviceEK(context.Background(), tc.G)
+	require.NoError(t, err)
+
+	fetchedDevices, err := GetOwnDeviceEKs(context.Background(), tc.G)
+	require.NoError(t, err)
+
+	require.Equal(t, 1, len(fetchedDevices))
+	require.Equal(t, metadata, fetchedDevices[0])
```

Comment:
let's verify the generation number, once we do the storage half we can verify beyond the hardcoded value !